examples/kotlinExample/src/main/kotlin/io/realm/examples/kotlin/KotlinExampleActivity.kt

```diff
@@ -30,11 +30,10 @@ import org.jetbrains.anko.async
 import org.jetbrains.anko.uiThread
 import kotlin.properties.Delegates
 
-
 class KotlinExampleActivity : Activity() {
 
     companion object {
-        val TAG: String = KotlinExampleActivity::class.qualifiedName as String
+        val TAG: String? = KotlinExampleActivity::class.java.simpleName
```

Comment:
the class name can't be null? unless there's something wrong with the classloader?
`val TAG: String = KotlinExampleActivity::class.java.simpleName` instead?